include/libdnf/rpm/package_query.hpp

```diff
@@ -587,6 +587,10 @@ class PackageQuery : public PackageSet {
     // TODO(dmach): consider removing the installed packages during the filtering
     void filter_priority();
 
+    /// Filter packages, which are installed but not available in any enabled repository.
```

Comment:
What about to replace `Filter packages` by `Keep packages`?
May I also ask you to describe the behavior with regular excluded packages?

Reply:
Sure. Done.